test/test.js

```diff
@@ -774,3 +774,12 @@ test('fail-fast mode', function (t) {
 		t.end();
 	});
 });
+
+test('power-assert support', function (t) {
+	t.plan(2);
+
+	execCli('fixture/power-assert.js', function (err, stdout, stderr) {
+		t.ok(err);
+		t.true(stderr.indexOf('t.ok(a === \'bar\')\n') !== -1);
```

Comment:
How is this testing that it's working? It's only the normals output.

And should be:

````js
t.not(stderr.indexOf('t.ok(a === \'bar\')\n'), -1);
```